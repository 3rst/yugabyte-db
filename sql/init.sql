\set ECHO none
set client_min_messages TO error;
\i orafce.sql
set client_min_messages TO default;