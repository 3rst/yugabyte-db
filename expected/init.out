\set ECHO none
psql:orafunc.sql:1946: NOTICE:  type "varchar2" is not yet defined
DETAIL:  Creating a shell type definition.
psql:orafunc.sql:1953: NOTICE:  argument type varchar2 is only a shell
psql:orafunc.sql:1967: NOTICE:  return type varchar2 is only a shell
psql:orafunc.sql:1974: NOTICE:  argument type varchar2 is only a shell
psql:orafunc.sql:1995: NOTICE:  argument type varchar2 is only a shell
psql:orafunc.sql:1995: NOTICE:  return type varchar2 is only a shell
psql:orafunc.sql:2121: NOTICE:  type "nvarchar2" is not yet defined
DETAIL:  Creating a shell type definition.
psql:orafunc.sql:2128: NOTICE:  argument type nvarchar2 is only a shell
psql:orafunc.sql:2142: NOTICE:  return type nvarchar2 is only a shell
psql:orafunc.sql:2149: NOTICE:  argument type nvarchar2 is only a shell
psql:orafunc.sql:2170: NOTICE:  argument type nvarchar2 is only a shell
psql:orafunc.sql:2170: NOTICE:  return type nvarchar2 is only a shell
